### PR TITLE
PPTT-2708 Added installation guide

### PR DESCRIPTION
**Pull request by azarecki**

### CODE REVIEW

File: data_collection/web/advanced.rst [hunk 1]
```diff
@@ -4,6 +4,53 @@
...
+        _paq.push(["setTrackerUrl", u+"ppms.php"]);
+        _paq.push(["setSiteId", "XXX-XXX-XXX-XXX-XXX"]);
+        var d=document, g=d.createElement("script"), s=d.getElementsByTagName("script")[0];
+        g.type="text/javascript"; g.async=true; g.defer=true; g.src=u+"ppms.js"; s.parentNode.insertBefore(g,s);
+      })();
+    </script>
+
+This code initializes the Analytics tracker in following ways:
+
+    #. Initializes the global ``_paq.push`` command queue that schedules commands to be run when the Analytics tracker library
```
Comment (adaqus) [inline data_collection/web/advanced.rst:46]:
```suggestion
    #. Initializes the global ``_paq`` command queue that schedules commands to be run when the Analytics tracker library
```

File: data_collection/web/advanced.rst [hunk 2]
```diff
@@ -24,3 +71,81 @@ The following global names are used by PPAS JavaScript tracker. Websites that wi
...
+    _paq.push(["setSiteId", "App2"]);
+
+    _paq.push(["setCookiePath", "/data/something_useful"]);
+
+    _paq.push(["setDomains", "example.com/data/something_useful"]);
+
+    _paq.push(["setTrackerUrl", u+"ppms.php"]);
+    _paq.push(["trackPageView"]);
+
+In this way, all actions tracked on ``/data/something_useful`` will be tracked for ``App2`` instead of ``App1``.
```
Comment (adaqus) [inline data_collection/web/advanced.rst:133]:
```suggestion
This way, all actions tracked on ``/data/something_useful`` will be tracked for ``App2`` instead of ``App1``.
```

**Review by wronan — APPROVED** [review]

File: data_collection/web/advanced.rst [hunk 3]
```diff
@@ -24,3 +71,81 @@ The following global names are used by PPAS JavaScript tracker. Websites that wi
...
+To set up tracking between multiple domains, you must use multiple functions ``setDomains`` to set a list of domains and
+``enableCrossDomainLinking`` to enable cross domain linking::
+
+    _paq.push(["setDomains", domains]);
+
+.. describe:: domains
+
+    **Required** ``array`` Domains array, with wildcards
+
+::
```
Comment (prk3) [inline data_collection/web/advanced.rst:109]: Description can be removed. We'll link setDomains name to JS tracking client api doc.

Comment (prk3) [inline data_collection/web/advanced.rst:109]: Description can be removed. We'll link setDomains name to JS tracking client api doc.

File: data_collection/web/advanced.rst [hunk 4]
```diff
@@ -24,3 +71,81 @@ The following global names are used by PPAS JavaScript tracker. Websites that wi
...
+To set up page performance metrics gathering use the
+``setTimingDataSamplingOnPageLoad`` function::
+
+    _paq.push(["setTimingDataSamplingOnPageLoad", updateTimingDataOnPageLoadSampling]);
+
+.. describe:: updateTimingDataOnPageLoadSampling
+
+    **Required** ``integer`` Value between 1 and 100 describing the percentage for data sampling
+
+::
```
Comment (prk3) [inline data_collection/web/advanced.rst:149]: Same here, we should give an example of usage and give a link to the function in API doc.

File: data_collection/web/guides.rst [hunk 5]
```diff
@@ -1,10 +1,21 @@
 Guides
 ======
 
-Instalation
------------
+Installing Tracking code
+------------------------
 
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop p
+This is the easiest and recommended way of tracking code installation. When Tag Manager is added to the site - it automatically publishes tracking code (using "Piwik PRO Analytics template").
```
Comment (prk3) [inline data_collection/web/guides.rst:7]:
```suggestion
Using tag manager's snippet is the easiest and recommended way of installing tracking code. When Tag Manager is added to the site - it automatically starts tracking user actions (using "Piwik PRO Analytics template").
```

File: data_collection/web/guides.rst [hunk 6]
```diff
@@ -1,10 +1,21 @@
...
+If you do not have Tag Manager on your website yet, follow this procedure to install it:
+
+#. Sign in to your PPAS with your admin or Super User account.
+#. Click on the menu button on the top left.
+#. Click on the "Websites" position.
+#. Choose the website for which you want to implement a tracking code.
+#. Select the "Installation" tab.
+#. The Tag Manager code snippet for your website is displayed under the "Website code for asynchronous tags" or "Website code for synchronous tags".
+
+In case you do not want to install Tag Manager on your website you can install tracking code via JavaScript Tracking Client snippet. Guide how to do it is available here: :ref:`data-collection-installing-tracking-code-via-snippet`
```
Comment (prk3) [inline data_collection/web/guides.rst:18]:
```suggestion
In case you do not want to install Tag Manager on your website, you can install tracking code via JavaScript Tracking Client snippet. Guide how to do it is available here: :ref:`data-collection-installing-tracking-code-via-snippet`
```

File: data_collection/web/advanced.rst [hunk 7]
```diff
@@ -4,6 +4,53 @@
 Advanced
 ========
 
+.. _data-collection-installing-tracking-code-via-snippet:
```
Comment (prk3) [inline data_collection/web/advanced.rst:7]: Let's move installation article to a separate document. Advanced would otherwise contain only this section (reserved-names have been moved to JS docs in a separate PR).

**Review by prk3 — CHANGES_REQUESTED** [review]
